i6_models/parts/conformer/mhsa.py

```diff
@@ -39,18 +39,28 @@ def __init__(self, cfg: ConformerMHSAV1Config):
         )
         self.dropout = cfg.dropout
 
-    def forward(self, input_tensor: torch.Tensor, sequence_mask: Optional[torch.Tensor] = None) -> torch.Tensor:
+    @staticmethod
+    def _invert_sequence_mask(sequence_mask: torch.Tensor):
+        """
+        Helper function to decide how to invert the sequence mask. For ONNX export use XOR with 1 since logical_not is not implemented.
+        Else logical_not is applied for efficiency reasons.
+
+        :param sequence_mask: bool mask of shape (B, T) to be inverted.
+        """
+        if torch.onnx.is_in_onnx_export():
+            return torch.logical_xor(sequence_mask, torch.ones(sequence_mask.shape, device=sequence_mask.device))
```

Comment:
```suggestion
            return torch.logical_xor(sequence_mask, torch.ones_like(sequence_mask))
```

Reply:
I think device is required. At least for me it crashed in my models when this was not set.